src/exe/main.cc

```diff
@@ -171,7 +171,7 @@ DDRes get_library_path(TempFileHolder &libdd_profiling_path,
 
 DDRes check_incompatible_options(const DDProfContext &ctx) {
   if (context_allocation_profiling_watcher_idx(ctx) != -1 && ctx.params.pid &&
```

Comment:
CLI has something to manage incompatibilities. The only part we can not check is if the allocation profiling is not active.
Which makes my comment only half useful ^^